.changeset/stale-monkeys-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+fix: not being able to access or create rooms with join code
```

Comment:
This message should be user friendly, check it out the docs and edit, please
https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#4.-adding-changeset-to-your-pull-request

Reply:
thanks for the heads up!